src/components/IndividualArticle.jsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from "react";
+import { getArticleByID } from "../../api.js";
+import { useParams } from "react-router-dom";
+import { useContext } from "react";
+import { CurrentPageLabelContext } from "../contexts/CurrentPageLabel.jsx";
+
+// {/* <p>Individual Article here, article is {article.title}</p>; */}
+
+function IndividualArticle() {
+  const { currentPageLabel, setCurrentPageLabel } = useContext(
+    CurrentPageLabelContext
+  );
+
+  const [article, setArticle] = useState([]);
+  const { article_id } = useParams();
+
+  setCurrentPageLabel(article.title);
+
+  useEffect(() => {
+    getArticleByID(article_id).then((response) => {
+      //
+      console.log(response.data.article);
+      setArticle(response.data.article[0]);
+    });
+  }, []);
+
+  return (
+    <>
+      <div className="full-individual-article">
+        <div className="full-individual-article-itself">
+          <div className="full-individual-article-things-to-center">
+            {currentPageLabel ? (
+              <h2 className="current-page-label">{currentPageLabel}</h2>
+            ) : (
```

Comment:
There are quite a few div elements in your code, some of which could be replaced with more semantic HTML tags. Here is a link from the notes for more information: [https://www.semrush.com/blog/semantic-html5-guide/](url)